src/v2/Components/v2/ArtworkFilter/ArtworkFilters/SizeFilter2.tsx

```diff
@@ -128,6 +128,13 @@ export const SizeFilter2: React.FC<SizeFilter2Props> = ({ expanded }) => {
     v3: { my: 1, secondaryVariant: "xs" as TextVariant },
   })
 
+  useEffect(() => {
+    // if filter state is being reset, then also clear local input state
+    if (reset) {
+      setCustomSize({ height: ["*", "*"], width: ["*", "*"] })
+    }
+  }, [reset])
+
```

Comment:
Tried my best to get a Jest/Enzyme spec around this but had to throw in the towel. 

Couldn't get farther than this assertion no matter what I threw at it 🤯 

```ts
const wrapper = getWrapper()

// set and confirm filter pre-state
context.setFilter("width", "1-2")
context.setFilter("height", "3-4")
expect(context.filters.width).toEqual("1-2")
expect(context.filters.height).toEqual("3-4")

// reveal custom size
wrapper.find("button").filterWhere(n => n.text() === "Show custom size").simulate("click")
await flushPromiseQueue()
wrapper.update()

// expect inputs to be prepopulated
const heightMinValue = wrapper.find(`input[name='height_min']`).instance().value

// 🤯 🤯 🤯
expect(heightMinValue).toEqual("1") // FAILS, but works IRL
```

Reply:
We've seen several examples of this with the filters (for some reason?), Enzyme tests that seem to just not work as expected, despite our attempts to get them working, which we then punt on.